DcCore/DcCore/Helper/DatabaseHelper.swift

```diff
@@ -14,12 +16,7 @@ public class DatabaseHelper {
     }
 
     var localDbFile: String {
-        return localDocumentsDir.appendingPathComponent("messenger.db").path
-    }
-
-    var localDocumentsDir: URL {
-        let paths = NSSearchPathForDirectoriesInDomains(.libraryDirectory, .userDomainMask, true)
-        return URL(fileURLWithPath: paths[0], isDirectory: true)
+        return FileManager.default.localDocumentsDir.appendingPathComponent("messenger.db").path
```

Comment:
this unrelated change,  touching a _very_ sensible area,  is no longer needed. i think, the additional abstraction is not really helpful without code being reused.

but no blocker.